.github/pull_request_template.md

```diff
@@ -13,6 +13,13 @@ resolves #
   understand your changes. Include any tradeoffs you considered.
 -->
 
+### Interface Changes
+
+<!---
+  Describe the changes made in this PR to any user interfaces, and their
+  expected impact on users. 
```

Comment:
Does this include all of:
- dbt developers, writing code in their projects
- dbt developers, invoking dbt commands/params on the CLI
- consumers of dbt metadata (logs + JSON artifacts)
- maintainers of adapter plugins